packages/tables/src/elements/OverflowButton.tsx

```diff
@@ -5,57 +5,27 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { ButtonHTMLAttributes, useState, forwardRef } from 'react';
-import PropTypes from 'prop-types';
-import { composeEventHandlers } from '@zendeskgarden/container-utilities';
-import OverflowStrokeIcon from '@zendeskgarden/svg-icons/src/16/overflow-stroke.svg';
-import { StyledOverflowButton, StyledOverflowButtonIconWrapper } from '../styled';
+import React, { ButtonHTMLAttributes, forwardRef } from 'react';
+import OverflowStrokeIcon from '@zendeskgarden/svg-icons/src/16/overflow-vertical-stroke.svg';
+import { StyledOverflowButton } from '../styled';
 import { useTableContext } from '../utils/useTableContext';
 
-export interface IOverflowButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
-  /** @ignore Applies hover styling */
-  isHovered?: boolean;
-  /** @ignore Applies active styling */
-  isActive?: boolean;
-  /** @ignore Applies focus styling */
-  isFocused?: boolean;
-}
-
 /**
  * @deprecated use `Table.OverflowButton` instead
  *
  * @extends ButtonHTMLAttributes<HTMLButtonElement>
  */
-export const OverflowButton = forwardRef<HTMLButtonElement, IOverflowButtonProps>(
-  ({ onFocus, onBlur, isFocused: focused, ...other }, ref) => {
-    const [isFocused, setIsFocused] = useState(false);
-    const { size } = useTableContext();
+export const OverflowButton = forwardRef<
+  HTMLButtonElement,
+  ButtonHTMLAttributes<HTMLButtonElement>
+>((props, ref) => {
+  const { size } = useTableContext();
 
-    return (
-      <StyledOverflowButton
-        onFocus={composeEventHandlers(onFocus, () => {
-          setIsFocused(true);
-        })}
-        onBlur={composeEventHandlers(onBlur, () => {
-          setIsFocused(false);
-        })}
-        size={size}
-        isFocused={typeof focused === 'undefined' ? isFocused : focused}
-        ref={ref}
-        {...other}
-      >
-        <StyledOverflowButtonIconWrapper>
-          <OverflowStrokeIcon />
-        </StyledOverflowButtonIconWrapper>
-      </StyledOverflowButton>
-    );
-  }
-);
+  return (
+    <StyledOverflowButton type="button" size={size} ref={ref} {...props}>
```

Comment:
Yep, good call!

Reply:
Ah, small correction: `size` is definitely needed for calculating the margin / centering the button in the row. This can be set as a transient prop though to prevent DOM leaking.